src/pages/settings/Security/TwoFactorAuth/TwoFactorAuthForm/index.android.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import BaseTwoFactorAuthForm from './BaseTwoFactorAuthForm';
+
+type TwoFactorAuthFormProps = {
```

Comment:
this is a module which contain platform specifc files so you should create seprate `types.ts` file and put types there